src/build.d

```diff
@@ -1135,7 +1111,11 @@ Params:
 auto runCanThrow(T)(T args)
 {
     auto res = run(args);
-    enforce(!res.status, res.output);
```

Comment:
Using `enforce` here is confusing the to the developer because now you get a stack-trace for `build.d` and then the actual error message you're interested in.

Example:
```
object.Exception@build.d(1198): /media/disk1/braddr/sandbox/at-client/pull-3708668-Linux_32/dmd/generated/linux/release/32/examples/impvisitor.o: In function `_D3dmd5parse__T6ParserTSQw7astbase7ASTBaseZQBc13parseDeclDefsMFiPCQCmQBrQBm7DsymbolPSQDfQDe__T16PrefixAttributesTQDmZQxZPSQEq4root5array__T5ArrayTQDdZQl':
/media/disk1/braddr/sandbox/at-client/pull-3708668-Linux_32/dmd/src/dmd/parse.d:584: undefined reference to `Array<ASTBase::Dsymbol*>::operator[](unsigned int)'
/media/disk1/braddr/sandbox/at-client/pull-3708668-Linux_32/dmd/src/dmd/parse.d:826: undefined reference to `Array<ASTBase::Dsymbol*>::operator[](unsigned int)'
/media/disk1/braddr/sandbox/at-client/pull-3708668-Linux_32/dmd/src/dmd/parse.d:845: undefined reference to `Array<ASTBase::Dsymbol*>::operator[](unsigned int)'
/media/disk1/braddr/sandbox/at-client/pull-3708668-Linux_32/dmd/src/dmd/parse.d:946: undefined reference to `Array<Identifier*>::operator[](unsigned int)'
/media/disk1/braddr/sandbox/at-client/pull-3708668-Linux_32/dmd/src/dmd/parse.d:965: undefined reference to `Array<ASTBase::Expression*>::operator[](unsigned int)'
...
```

In the above example we get a line-number in `build.d`, but we don't care about that line number, we really only care about the error message.

Reply:
Indeed, but what is most helpful is the actual command that failed.  But I think that's available in the verbose output, yes?

Reply:
Yeah `-v` will show you the command.  Most of the time you just want to see the error message though, as mostly it will just be compilation errors that point you to the files/locations you need to fix.